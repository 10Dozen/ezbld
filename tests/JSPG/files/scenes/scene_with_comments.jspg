# SceneWithComments

Line 1.

// Line commented.

Block 1.
Line 1-1.
// Line 1-2.
Line 1-3.

// *pre_exec: Foo.Bar()
**post_exec:
    // Foo.Bar()
    Bar.Foo()
*
