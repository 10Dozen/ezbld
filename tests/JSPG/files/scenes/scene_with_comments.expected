[0]
cls = JSPGScene
name = SceneWithComments | TEXT
post_exec =
    ()=>{
    \tBar.Foo()
    } | MULTILINE_FUNCTION

desc =
    "Line 1."

    `Block 1.<br>
    Line 1-1.<br>
    Line 1-3.<br>`
